app/client/components/delivery/records/deliveryRecords.tsx

```diff
@@ -292,102 +292,98 @@ export const DeliveryRecordsFC = (props: DeliveryRecordsFCProps) => {
               isGift={isGift(props.productDetail.subscription)}
             />
           </div>
-          {productType.delivery.records.canReportProblem &&
-            props.data.results.find(record => !record.problemCaseId) && (
-              <>
-                <h2
+          {props.data.results.find(record => !record.problemCaseId) && (
+            <>
```

Comment:
there's a lot of whitespace changes in this commit (thanks to `prettier`) so I advise you do this 
![image](https://user-images.githubusercontent.com/19289579/76880385-7c0b5f00-686f-11ea-8e22-eb416436f43e.png) to see the pertinent changes